hibike/devices/YogiBear/YogiBear.cpp

```diff
@@ -63,43 +64,19 @@ void loop() {
 
 uint32_t device_write(uint8_t param, uint8_t* data, size_t len) {
   switch (param) {
-    case ENABLE:
-      if (data[0] == 0) {
-        motorDisable();
-      } else {
-        motorEnable();
-      }
-      return sizeof(bool);
-      break;
-
-    case COMMAND_STATE: 
-      driveMode = data[0];
-
-      switch (driveMode) {
-        case MANUALDRIVE:
-          disablePID();
-          break;
-        case PID_VEL:
-          enableVel();
-          break;
-        case PID_POS:
-          enablePos();
-          break;
-        default:
-          driveMode = MANUALDRIVE;
-          disablePID();
-          break;
-      }
 
-      return sizeof(uint8_t);
-      break;
-
-    case DUTY_CYCLE: 
+    case DUTY_CYCLE:
+      motorEnable();
```

Comment:
if duty cycle is 0, do we catch that here? do we care about special casing 0, or does it just work fine?